core/shared/platform/windows/win_time.c

```diff
@@ -18,3 +68,126 @@ os_time_get_boot_microsecond()
 
     return ((uint64)ts.tv_sec) * 1000 * 1000 + ((uint64)ts.tv_nsec) / 1000;
 }
+
+uint64
+os_win_clock_res_get(__wasi_clockid_t clock_id, __wasi_timestamp_t *resolution)
```

Comment:
```suggestion
os_clock_res_get(bh_clock_id_t clock_id, uint64 *resolution)
```

And in addition to that, I'd define enum in the `platform_common.h` with all the clock IDs, something like:
```c
typedef enum {
  BH_CLOCK_REALTIME,
 ....
} bh_clock_id_t;
```